### PR TITLE
Completed prep-css-flexbox. Paul Son

### PR DESCRIPTION
**Pull request by contactpaulson**
prep-css-flexbox display-flex
prep-css-flexbox justify-content
)
prep-css-flexbox align-items

### CODE REVIEW

File: prep-css-flexbox/align-items.html [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <meta http-equiv="X-UA-Compatible" content="ie=edge" />
+    <title>CSS Flexbox - align-items</title>
+    <link rel="stylesheet" href="reset.css" />
+    <link rel="stylesheet" href="starwars.css" />
+    <style>
+      /* your styles here 👇 */
+      #card-deck {
+        display: flex;
+        flex-basis: calc(100% / 3);
```
Comment (thebearingedge) [inline prep-css-flexbox/align-items.html]: Careful `flex-basis` goes on flex items (children of elements with `display: flex`). You can add another rule set with a selector for `.card-container` and apply the `flex-basis` there.

File: prep-css-flexbox/justify-content.html [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <meta http-equiv="X-UA-Compatible" content="ie=edge" />
+    <title>CSS Flexbox - justify-content</title>
+    <link rel="stylesheet" href="reset.css" />
+    <link rel="stylesheet" href="starwars.css" />
+    <style>
+      /* your styles here 👇 */
+      #card-deck {
+        display: flex;
+        flex-basis: 25%;
```
Comment (thebearingedge) [inline prep-css-flexbox/justify-content.html]: Same here, try putting the `flex-basis` on the `.card-container`.

**Review by thebearingedge — CHANGES_REQUESTED** [review]
great, just a couple small changes

**Comment by contactpaulson** [comment]
prep-css-flexbox display-flex
prep-css-flexbox justify-content
)
prep-css-flexbox align-items

**Comment by contactpaulson** [comment]
prep-css-flexbox display-flex
prep-css-flexbox justify-content
)
prep-css-flexbox align-items

**Review by thebearingedge — APPROVED** [review]
got it!

**Pull request merged**